### PR TITLE
Image analysis api with example

### PR DESCRIPTION
**Pull request by jctilghman**

### CODE REVIEW

File: image-analysis/api/image-analysis-swagger.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+#                                   Parameters                                 #
+################################################################################
+
+parameters:
+
+  image:
+    name: image
+    in: body
+    description: |
+      The biometric image that is being submitted for analysis.
```
Comment (JHowix) [inline image-analysis/api/image-analysis-swagger.yml]: need to include that its a PNG and base64 encoded:

"biometric image data that is being submitted for analysis, in PNG format, encoded as a base64 string"

File: image-analysis/api/image-analysis-swagger.yml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+# format of the responses to the client (Accepts)
+produces:
+  - application/json
+  
+################################################################################
+#                                   Tags                                       #
+################################################################################
+
+tags:
+  - name: Biometric Operations
```
Comment (JHowix) [inline image-analysis/api/image-analysis-swagger.yml]: would change this to "Analysis Operations"

File: image-analysis/api/image-analysis-swagger.yml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    type: string
+    description: |
+      Relevant and concise diagnostic information in the event of a client side error (e.g. malformed requests, or invalid image encoding).
+    example: Unable to decode image data as a PNG.
+
+  ServerError:
+    type: string
+    description: |
+      Relevant and concise diagnostic information in the event of a server side error (e.g. invalid license, or failure to generate a template).
+    example: The internal license has expired.
```
Comment (JHowix) [inline image-analysis/api/image-analysis-swagger.yml]: lets change this error -- I dont want people running around with the idea they have to license these things -- maybe "array out of bounds" or something generic

File: image-analysis/api/README.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
+<!DOCTYPE html>
```
Comment (JHowix) [inline image-analysis/api/README.md]: I think this file extension is wrong -- this is html?

Comment (jctilghman) [reply image-analysis/api/README.md]: This was in the existing READMEs too. I think GitHub might like READMEs to be .md files even if they are actually html. Maybe it won't display properly with an html extension.

File: image-analysis/api/image-analysis-swagger.yml (hunk 3 above)
Comment (jctilghman) [reply image-analysis/api/image-analysis-swagger.yml]: I'll use "Error encoding response" from the example implementation.

**Pull request merged**